test/tic_tac_toe/ai_test.clj

```diff
@@ -192,4 +194,49 @@
   (let [player  (->AiPlayer "o")
         game { :board [["_" "_" "_" ]["_" "_" "_" ]["_" "_" "_" ]] :ai-marker "o" :player-marker "x"}]
     (testing "creates defrecord of player protocol"
-      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
\ No newline at end of file
+      (is (= (assoc game :board [["o" "_" "_"] ["_" "_" "_"] ["_" "_" "_"]]) (next-move player game))))))
+
+(defn win-or-draw? [player-marker winner-state]
+ (or (= player-marker winner-state) (= player-marker "its a draw")))
+
+(defn correct-ai-choice? [game]
+  (let[winner (winner? (:board game))]
```

Comment:
Also, I believe I have mentioned this before, but having a function with a question mark, implies the function returns a boolean. This may work in cases where you are using truthy falsey assertions, which you shouldn't be doing anyways, but will be very confusing to someone using your code.